components/NavigationBar/src/MDCNavigationBar.m

```diff
@@ -372,12 +372,11 @@ - (void)layoutSubviews {
 
 - (CGSize)sizeThatFits:(CGSize)size {
   CGSize intrinsicContentSize = [self intrinsicContentSize];
-  return CGSizeMake(size.width, intrinsicContentSize.height);
+  return CGSizeMake(size.width, MIN(size.height, intrinsicContentSize.height));
```

Comment:
I'm not sure if any clients are using sizeThatFits: on MDCNavigationBar or MDCButtonBar with CGSizeZero. In that case, this change can break them. Maybe I should guard this behind size.height > 0?

Reply:
In order to maintain the existing functionality it may be sensible to have a conditional clause for the empty size case where we use the old behavior.

Reply:
Done.